main.cc

```diff
@@ -116,6 +121,14 @@ static void on_session_error(otc_session *session,
   std::cout << "Session error. Error : " << error_string << std::endl;
 }
 
+static void on_session_reconnection_started(otc_session *session, void *user_data) {
```

Comment:
@robjperez I am not getting these callbacks, what I am missing?, in c++ do I need to add a listener like in Android?

Reply:
you should get them, you already added the functions to the callback struct